feature-onboarding-impl/src/main/java/io/novafoundation/nova/feature_onboarding_impl/domain/OnboardingInteractorImpl.kt

```diff
@@ -12,7 +14,9 @@ class OnboardingInteractorImpl(
     }
 
     override suspend fun isCloudBackupAvailableForImport(): Boolean {
-        return !cloudBackupService.session.isSyncWithCloudEnabled()
+        val syncWithCloudNotEnabled = !cloudBackupService.session.isSyncWithCloudEnabled()
+        val withoutAddedAccounts = !accountRepository.hasActiveMetaAccounts()
```

Comment:
Why do we limit it like that? It should still be possible to import from backup even when we already have accounts added. We specifically designed a sync strategy for it to merge local and cloud accoutns in a way suitable for import

Reply:
I followed our design layouts. Apparently, if we have added wallets, we can only enable backup from the settings

Reply:
From personal messages: we return such a possibility as it was confirmed on the planning